boilerplates/react-telefunc/package.json

```diff
@@ -36,7 +36,12 @@
         "$all": [
           "react",
           "telefunc"
-        ]
+        ],
+        "$not": {
+          "$in": [
+            "drizzle"
+          ]
+        }
```

Comment:
Should not be necessary anymore

Reply:
I'm sorry, but I'm not sure about that, I've tried `pnpm run cli --react --fastify --drizzle --telefunc` with latest commits without these lines, the result is mixed between `react-telefunc` and `drizzle` files, and `react-telefunc`'s files override `drizzle`'s files (e.g. the `+data.ts` file contains codes from `react-telefunc` only).